src/test/java/org/dependencytrack/notification/publisher/AbstractPublisherTest.java

```diff
@@ -85,7 +88,7 @@ public abstract class AbstractPublisherTest<T extends Publisher> extends Persist
     }
 
     @Test
-    public void testInformWithBomConsumedNotification() {
+    public void testInformWithBomConsumedNotification(WireMockRuntimeInfo wmRuntimeInfo) {
```

Comment:
Is there a way to have the WireMock instance as a class-level field, instead of "colouring" all methods by requiring a `WireMockRuntimeInfo` parameter?